### DISCUSSION TITLE
popey snaps - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#54] [General]
# popey snaps - Alan Pope's blog

popeys snap status

https://popey.com/blog/snaps/

**Comment by LorenDB** [comment]
Hey, this page keeps moving to the top of my RSS feed. Is there any way you could prevent that from happening? Likely your site is set up to order posts by last modified, so this one is constantly jumping to the top. Maybe you could tweak the Hugo RSS template to manually exclude this post from that sorting?

**Reply by popey** [reply]
Oof! Sorry about that. I didn't realise these pages were included in the RSS feed. Apologies.
I have hard-wired the date on this page to be a week ago, which should drop it down the list.
Thanks for letting me know!

**Reply by LorenDB** [reply]
tysm! Probably it's partly because I subscribed to the /index.xml feed which gives a feed of the whole site, but then again, I do like to see random pages like this that aren't necessarily blog posts...
